pkgpanda/cli.py

```diff
@@ -83,6 +84,9 @@ def uninstall(install, repository):
     # Removing /opt/mesosphere
     check_call(['rm', '-rf', install.root])
 
+    if os.path.exists(install.sysctl_config):
```

Comment:
No point in checking for os.path.exists, the `-f` in the `rm` should be plenty good.